### DISCUSSION TITLE
Tailwind broken after ng update

### DISCUSSION THREAD

**Discussion by nathan30** [tailwindlabs/tailwindcss#4178] [Help]
### What version of Tailwind CSS are you using?

2.1.1

### What build tool (or framework if it abstracts the build tool) are you using?

custom-webpack : 11.0.0

### What version of Node.js are you using?

v14.16.1

### What browser are you using?

Firefox

### What operating system are you using?

Ubuntu 20.10

### Reproduction repository

https://github.com/nathan30/tailwind_bug

### Describe your issue

After update Angular from 11.0.4  to 11.2.10 all my tailwind doesn't work anymore. The class don't work, the image path I put in the tailwind.config.js don't work anymore

Thanks

**Comment by adamwathan** [comment]
Hey! This sounds like an Angular related issue rather than a Tailwind issue, as we don't have any Angular-specific code or anything. Going to move to discussions in case someone else has any help to offer.
